Tooling phoenix

usirin opened the pull request:
Update eslint, tsconfig & prettier.

Ketcap commented:
lgtm! 🤣
Huge pr to look out. I'm assuming all the changes are done by linter / automation so should be good to go.

usirin commented:
Yeah sorry :D 

so here is what i did:
- i moved the packages/apps we don't use anymore to `deprecated` folder
  - `apps/pano`
  - `apps/home`
  - `packages/kampus-ui`
  - `packages/pano-button`

- i removed `packages/kampus-eslint-config` and `packages/kampus-tsconfig`
- created `config` folder and added it as workspaces folder in `package.json`. I thought we can host the configs over there.
- redid the prettier config based on https://github.com/t3-oss/create-t3-turbo/blob/main/prettier.config.cjs
